usecases/MinimumWorkingExample/upload_scripts/doit_upload.py

```diff
@@ -478,6 +470,10 @@ def _mixture_upload(
         parents=list(mixture_parent_ingredients.keys()),
     )
 
+    MixtureModel = o.generate_typechecker(mixture_sample_type)
+    mixture_model_return = MixtureModel(**mixture_metadata_dict)
+    mixture_metadata_dict = mixture_model_return.dict(exclude_unset=True)
```

Comment:
Please add the use of the typechecker in teh readme of "interbis modul"

Reply:
I have extended the readme. Can you check it if it is enough for the potential users?